pkg/services/ngalert/api/api_alertmanager_test.go

```diff
@@ -226,26 +139,6 @@ func TestAlertmanagerConfig(t *testing.T) {
 			body := asGettableUserConfig(t, response)
 			require.Equal(t, apimodels.Provenance(ngmodels.ProvenanceAPI), body.AlertmanagerConfig.Route.Provenance)
 		})
-		t.Run("contact point from GET config has expected provenance", func(t *testing.T) {
```

Comment:
Let's keep this test for now by creating a drop-in replacement for `sut.RoutePostAlertingConfig`:

```go
// RoutePostAlertingConfig drop-in replacement for removed POST endpoint to make test transition easier. 
func RoutePostAlertingConfig(t *testing.T, mam *notifier.MultiOrgAlertmanager, rc *contextmodel.ReqContext, amConfig string) {
	t.Helper()
	cfg := apimodels.PostableUserConfig{}
	err := json.Unmarshal([]byte(amConfig), &cfg)
	require.NoError(t, err)
	err = mam.SaveAndApplyAlertmanagerConfiguration(rc.Req.Context(), 1, cfg)
	require.NoError(t, err)
}
```

used like:

```go
RoutePostAlertingConfig(t, sut.mam, rc, validConfig)
```